docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +992,154 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems can be accomplished by
+    externalizing the LDAP Active Directory Security Manager administrator
+    account credentials using Kubernetes and Hashicorp Vault secrets. To use
+    these externalized credentials, as prerequisite should be familiar setting
+    up Kubernetes secrets and/or Hashicorp Vault secrets.</para>
+
+    <para>The LDAP Administrator account is the only HPCC user that must have
+    Active Directory administrator rights. This account should exist in the
+    configured "systemBasedn" branch of the Active Directory, typically set to
+    cn=Users.</para>
+
+    <sect2 id="Configuring_For_KubernetesSecrets">
+      <title>Configuring LDAP for Kubernetes</title>
+
+      <para>To configure secrets in Kubernetes, from the Kubernetes CLI,
```

Comment:
Technically you are creating a secret, not configuring secrets. And, you are executing a Kubernetes command from a terminal, not an actual Kubernetes CLI. Consider something like:

To create a Kubernetes secret for the HPCC Admin account, execute the following command:

<command>

Where:
    * describe each value the user provides
    
    
To view the secret just created, execute the following command:

kubectl get secret ....